lesson.txt

```diff
@@ -0,0 +1,2 @@
+git checkout -b iss12  ---make working branch
```

Comment:
next time do not add your personal files in your commit. You may use `git add -i` and select the files that need to be committed